build.cake

```diff
@@ -125,6 +125,7 @@ Task("Run-Unit-Tests")
                 parameters.Paths.Files.TestCoverageOutputFilePath,
                 new OpenCoverSettings {
                     ReturnTargetCodeOffset = 0,
+                    Register = "user",
```

Comment:
Isn't user default though?
https://github.com/cake-build/cake/blob/develop/src/Cake.Common/Tools/OpenCover/OpenCoverSettings.cs#L74

Think issue could actually be that server is 32 bit and need to specify Path32-